include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -251,12 +314,14 @@ class operational_domain_impl
      * @param st Statistics of the process.
      */
     operational_domain_impl(const Lyt& lyt, const std::vector<TT>& tt, const operational_domain_params& ps,
-                            operational_domain_stats& st) noexcept :
+                            operational_domain_stats& st, bool consider_logic = true) noexcept :
```

Comment:
If you move to another header, you could consider, adding another constructor here, too. Explicitly for your functionality

Reply:
Great idea. Thank you!